protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -5009,3 +5016,38 @@ message AMMEdge {
   // Cursor that can be used to fetch further pages.
   string cursor = 2;
 }
+
+message EstimateAMMBoundsRequest {
+  // Base price of the AMM pool, the price is an integer, for example `123456` is a correctly
+  // formatted price of `1.23456` assuming market configured to 5 decimal places.
+  string base_price = 1;
+  // Upper price of the AMM pool, the price is an integer, for example `123456` is a correctly
+  // formatted price of `1.23456` assuming market configured to 5 decimal places.
+  string upper_price = 2;
+  // Lower price of the AMM pool, the price is an integer, for example `123456` is a correctly
+  // formatted price of `1.23456` assuming market configured to 5 decimal places.
+  string lower_price = 3;
+  // Leverage at the upper price of the AMM pool.
+  string leverage_at_upper_price = 4;
```

Comment:
This and `leverage_at_lower_price` should also be optional, and if they aren't provided we just ust the market risk factors.

Reply:
done